qa/tasks/vstart_runner.py

```diff
@@ -943,7 +944,11 @@ def run(self, *args, **kwargs):
 
 class LocalContext(object):
     def __init__(self):
-        self.config = {'cluster': 'ceph'}
+        cluster_name = 'ceph'
+        self.config = {'cluster': cluster_name}
+        self.ceph = {}
+        self.ceph[cluster_name] = Namespace()
```

Comment:
line 949 and 950 can be combined to `self.ceph = {cluster_name: Namespace()}`

Reply:
True.